InfiniteBlanK3T(2).CodingTracker/MainCodingTracker/Program.cs

```diff
@@ -0,0 +1,139 @@
+    static void Main()
+    {
+        MainMenu();
+    }
+    static void MainMenu()
+    {
+       
+        CRUDController action = new();
+        UserInput input = new();
+        bool endApp = false;
+
+        while (!endApp)
+        {
+            
+            Console.Clear();
+            Console.WriteLine("\n-------------------------------\n");
+            Console.WriteLine("\tCODING TRACKER");
+            Console.WriteLine("\n-------------------------------\n");
+            Console.WriteLine($"Welcome, {database.Name.Replace("_", " ")}!. Today {(DateTime.Now).ToString("dd - MM - yyyy")}");
+            Console.WriteLine("\n0*. Start Tracking. . .");
+            Console.WriteLine("-------------------------------");            
+            Console.WriteLine("1. View Records.");
+            Console.WriteLine("2. Insert Records.");
+            Console.WriteLine("3. Delete Records.");
+            Console.WriteLine("4. Update Records.");
+            Console.WriteLine("5*. See Record Report.");
+            Console.WriteLine("-------------------------------");
+            Console.WriteLine("6. CREATE YOUR OWN TRACKER !");            
+            Console.WriteLine("-------------------------------\n");
+            Console.WriteLine("7. Exit.\n");
+            Console.Write("Your option: ");
+
+            var userOption = Console.ReadLine();
+            Console.Clear();
+
+            switch (userOption)
+            {
+                case "0":
+                    input.StartingTimerCount(database.Name);
+                    TaskComplete();                                        
+                    break;
+                case "1":
+                    action.GetAllRecords(database.Name);
+                    TaskComplete();
+                    break;
+                case "2":
+                    input.GetRecordFromUser(database.Name);
+                    TaskComplete();
+                    break;
+                case "3":
+                    action.Delete(database.Name);
+                    TaskComplete();
+                    break;
+                case "4":
+                    action.Update(database.Name);
+                    TaskComplete();
+                    break;
+                case "5":
+                    ReportTable(database.Name);
+                    TaskComplete();
+                    break;
+                case "6":
+                    database.Name = database.CreateNewRecord();                    
+                    TaskComplete();
+                    break;
+                case "7":
+                    Console.WriteLine("\nGoodbye!\n");
+                    Environment.Exit(0);
+                    break;
+                default:
+                    Console.Write("\nInvalid input! Please try again.");
+                    Console.ReadLine();
+                    break;
+            }
+        }
+    }
+
+    //Maybe a better way to do this but in the meantime this method looks good
```

Comment:
Have a solid think about your comments. Code should be self-explanatory. This one seems to be ok, since it's a reminder for your future self, but in general, the best you name your properties, methods, fields, arguments, the less you need to comment. 😊 Only keep comments that are absolutely necessary for your future self or other coders to understand what's going on and can't be explained by the code itself.